samples/sample_multi_transcode/src/pipeline_transcode.cpp

```diff
@@ -4499,27 +4420,31 @@ mfxStatus CTranscodingPipeline::Reset()
 mfxStatus CTranscodingPipeline::AllocAndInitVppDoNotUse(sInputParams *pInParams)
 {
     std::vector<mfxU32> filtersDisabled;
-    if(pInParams->DenoiseLevel==-1)
+    if (pInParams->DenoiseLevel == -1)
     {
-        filtersDisabled.push_back(MFX_EXTBUFF_VPP_DENOISE); // turn off denoising (on by default)
+        filtersDisabled.emplace_back(MFX_EXTBUFF_VPP_DENOISE); // turn off denoising (on by default)
```

Comment:
just note, no sense to switch to ```emplace_back``` here. For trivial types it brings no difference